src/main/scala/com/gu/sfl/controller/FetchArticlesController.scala

```diff
@@ -17,7 +17,7 @@ class FetchArticlesController(fetchSavedArticles: FetchSavedArticles)(implicit e
          syncedPrefs.savedArticles.map( sp =>
             logger.debug(s"Returning found: ${sp.articles.size} articles")
          )
-         Future { okSavedArticlesResponse(syncedPrefs) }
+         Future { okSycedPrefsResponse(syncedPrefs) }
```

Comment:
Given the change in response, should the controller be renamed to match (FetchArticlesController -> FetchSyncedPrefsController)